routing/validation_barrier.go

```diff
@@ -285,8 +285,6 @@ func (v *ValidationBarrier) SignalDependants(job interface{}, allow bool) {
 			delete(v.chanAnnFinSignal, msg.ShortChannelID)
 		}
 
-		delete(v.chanEdgeDependencies, msg.ShortChannelID)
```

Comment:
Nice catch! Is there a simple test case we can add here to protect against future regressions?

Reply:
Yes! Because a new validation barrier is added in gossip, the underlying logic is tested in that one. I think eventually we'll want to remove the usage of vb from router.

Reply:
can you explain this change in more detail?